.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "deps/emsdk"]
+	path = deps/emsdk
+	url = https://github.com/juj/emsdk.git
+[submodule "src/nn/wasm/src/external/tensorflow"]
+	path = src/nn/wasm/src/external/tensorflow
+	url = https://github.com/tensorflow/tensorflow.git
```

Comment:
I noticed that the commit of tensorflow you added is https://github.com/tensorflow/tensorflow/tree/04e8759ee2416baac1f31f6a27cb49a8b6051e19, any reasons to add this commit? Do we need to align to the version used by Android NN  or some releases of TensorFlow-Lite?

Reply:
No, we don't need to align the version. I used 
```
git submodule add https://github.com/tensorflow/tensorflow.git src/nn/wasm/src/external/tensorflow
```
And seems it fetch the newest version of tensorflow automatically.
Do I have a wrong use of `git submodule add`?